portfolio/src/main/webapp/index.html

```diff
@@ -129,8 +138,27 @@ <h3>SplitBill</h3>
             </div>
         </div>
         <div>
-            <p>This is my dog, Salem</p>
-            <img src='./images/salem.JPG' alt='Salem' width='500'/> 
+            <h2>Photography</h2>
+            <p>I started photography when I got my first camera back in 6th grade. What started with me taking random pictures of my dog turned into a huge hobby. I started doing free-lance photography for many small Instagram businesses and exploring other types of photography: street, landscape, macro, etc. </p>
```

Comment:
Nit: Freelance not free-lance.

(A nit is a short way of saying that the request change is a nitpick. It indicates that while the change would be nice to have, it's not important to the working of the code. Examples are things like common spelling or layout changes.)

Reply:
Got it, thank you for catching that!